packages/protobuf-test/src/create.test.ts

```diff
@@ -671,15 +671,14 @@ describe("create()", () => {
           // @ts-expect-error expected type error
           repeatedInt64JsStringField: [protoInt64.parse(6)],
           either: {
-            // @ts-ignore -- required for older TS
             case: "oneofInt64Field",
-            // @ts-ignore -- required for older TS
+            // @ts-expect-error expected type error
```

Comment:
In tests, we can replace some `@ts-ignore` annotations with `@ts-expect-error`, since all supported versions of TS produce an error here.